zubhub_frontend/zubhub/src/views/create_activity/script.js

```diff
@@ -10,787 +10,722 @@ const idPrefix = 'activitystep';
 const API = new ZubhubAPI();
 
 export const class_grades = [
-    { name: 'Grade 1-3', id: '1-3' },
-    { name: 'Grade 4-6', id: '4-6' },
-    { name: 'Grade 7-9', id: '7-9' },
-    { name: 'Grade 10-12', id: '10-12' },
+  { name: 'Grade 1-3', id: '1-3' },
+  { name: 'Grade 4-6', id: '4-6' },
+  { name: 'Grade 7-9', id: '7-9' },
+  { name: 'Grade 10-12', id: '10-12' },
 ];
 
 export const vars = {
-    image_field_touched: false,
-    video_field_touched: false,
-    upload_in_progress: false,
-    timer: { id: null },
-    default_state: {
-        loading: true,
-        error: null,
-        desc_input_is_focused: false,
-        video_upload_dialog_open: false,
-        select_video_file: false,
-        materials_used: [],
-        categories: [],
-        tag_suggestion: [],
-        tag_suggestion_open: false,
-        publish_types: [],
-        publish_visible_to_suggestion: [],
-        publish_visible_to_suggestion_open: false,
-        media_upload: {
-            upload_dialog: false,
-            images_to_upload: [],
-            videos_to_upload: [],
-            upload_info: {},
-            upload_percent: 0,
-            uploaded_images_url: [],
-            uploaded_videos_url: [],
-        },
+  image_field_touched: false,
```

Comment:
hey @coderatomy to make it easy, separate the linting and formatting from the changes.